src/sidebar.js

```diff
@@ -66,110 +66,109 @@ module.exports = {
-                "guides/social-signin/facebook",
-                "guides/social-signin/microsoft",
-                "guides/social-signin/github",
-                "guides/social-signin/apple",
-                "guides/social-signin/gitlab",
-                "guides/social-signin/auth0",
-                "guides/social-signin/slack",
-                "guides/social-signin/spotify",
-                "guides/social-signin/discord",
-                "guides/social-signin/twitch",
-                "guides/social-signin/netid",
-                "guides/social-signin/yandex",
-                "guides/social-signin/vk",
-                "guides/social-signin/dingtalk",
+              "Integrating providers": [
+                "kratos/social-signin/generic",
+                "kratos/social-signin/google",
+                "kratos/social-signin/facebook",
+                "kratos/social-signin/microsoft",
+                "kratos/social-signin/github",
+                "kratos/social-signin/apple",
+                "kratos/social-signin/gitlab",
+                "kratos/social-signin/auth0",
+                "kratos/social-signin/slack",
+                "kratos/social-signin/spotify",
+                "kratos/social-signin/discord",
+                "kratos/social-signin/twitch",
+                "kratos/social-signin/netid",
+                "kratos/social-signin/yandex",
+                "kratos/social-signin/vk",
+                "kratos/social-signin/dingtalk",
               ],
             },
-            "guides/social-signin/get-tokens",
-            "guides/social-signin/data-mapping",
+            "kratos/social-signin/get-tokens",
+            "kratos/social-signin/data-mapping",
           ],
         },
         {
           type: "category",
-          label: "Multi-Factor Authentication (MFA)",
+          label: "Multi-factor authentication (MFA)",
           items: [
             {
               type: "autogenerated",
-              dirName: "guides/mfa",
+              dirName: "kratos/mfa",
             },
           ],
         },
-        "guides/emails",
         {
           type: "category",
-          label: "Customizing Ory",
-          items: ["guides/custom-domains", "guides/bring-your-user-interface"],
+          label: "Passwordless",
+          items: ["kratos/passwordless/webauthn"],
         },
-      ],
-    },
-    {
-      Concepts: [
-        "concepts/terminology",
-        "concepts/project",
-        "concepts/redirects",
-        "concepts/personal-access-token",
-        "ecosystem/api-design",
-        "concepts/password-policy",
-      ],
-    },
-    {
-      type: "category",
-      label: "Identity Management (Ory Kratos)",
-      items: [
         {
-          Introduction: [
-            "kratos/index",
-            "kratos/quickstart",
-            "kratos/install",
-            "kratos/configuring",
+          type: "category",
+          label: "Sending emails",
```

Comment:
Any suggestions for the topic name? Maybe something that includes the "out-of-bounds" term? 
@aeneasr